sky/engine/core/dart/jni.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart_ui;
```

Comment:
Can you put this file in `sky/engine/bindings/jni` ?  It should eventually becomes its own built-in library.